dask_ml/wrappers.py

```diff
@@ -323,6 +323,12 @@ class Incremental(ParallelPostFit):
            a single NumPy array, which may exhaust the memory of your worker.
            You probably want to always specify `scoring`.
 
+    random_state : int or numpy.random.RandomState
+        Random object that determines how to shuffle blocks.
+
+    shuffle_blocks : bool
+        Whether to randomly shuffle the blocks or now
```

Comment:
typo: now -> not.

Could you elaborate on what this means? Specifically, clarify shuffling the block order from shuffling within blocks (from shuffling between blocks).